docs/static/setting-up-logstash.asciidoc

```diff
@@ -0,0 +1,183 @@
+[[setup-logstash]]
+== Setting Up and Running Logstash
+
+coming[5.0.0-alpha3, Includes breaking changes to the Logstash directory structure]
+
+Before reading this section, see <<installing-logstash>> for basic installation instructions to get you started. 
+
+This section includes additional information on how to set up and run Logstash, including:
+
+* <<dir-layout>>
+* <<config-setting-files>>
+* <<running-logstash>>
+
+[[dir-layout]]
+=== Logstash Directory Layout
+
+This section describes the default directory structure that is created when you unpack the Logstash installation packages.
+
+[[zip-targz-layout]]
+==== Directory Layout of `.zip` and `.tar.gz` Archives
+
+The `.zip` and `.tar.gz` packages are entirely self-contained. All files and
+directories are, by default, contained within the home directory -- the directory
+created when unpacking the archive.
+
+This is very convenient because you don't have to create any directories to start using Logstash, and uninstalling
+Lostash is as easy as removing the home directory.  However, it is advisable to change the default locations of the
+config and the logs directories so that you do not delete important data later on.
+
+[cols="<h,<,<m,<m",options="header",]
+|=======================================================================
+| Type | Description | Default Location | Setting
+| home
+  | Home directory of the Logstash installation.
+  | `{extract.path}`- Directory created by unpacking the archive
+  | `path.home`
+
+| bin
+  | Binary scripts, including `logstash` to start Logstash
+    and `logstash-plugin` to install plugins
+  | `{extract.path}/bin`
+ d|
+
+| settings
+  | Configuration files, including `logstash.yml` and `jvm.options`
+  | `{extract.path}/config`
+  | `path.settings`
+
+| logs
+  | Log files
+  | `{extract.path}/logs`
+  | `path.logs`
+
+| plugins
+  | Local, non Ruby-Gem plugin files. Each plugin is contained in a subdirectory. Recommended for development only.
+  | `{extract.path}/plugins`
+  | `path.plugins`
+
+|=======================================================================
+
+[[deb-layout]]
+==== Directory Layout of Debian and RPM Packages
+
+The Debian package and the RPM package each place config files, logs, and the settings files in the appropriate
+locations for the system:
+
+[cols="<h,<,<m,<m",options="header",]
+|=======================================================================
+| Type | Description | Default Location | Setting
+| home
+  | Home directory of the Logstash installation.
+  | `/usr/share/logstash`
+  | `path.home`
+
+| bin
+  | Binary scripts including `logstash` to start Logstash
+    and `logstash-plugin` to install plugins
+  | `/usr/share/logstash/bin`
+ d|
+
+| settings
+  | Configuration files, including `logstash.yml`, `jvm.options`, and `startup.options`
+  | `/etc/logstash`
+  | `path.settings`
+
+| conf
+  | Logstash pipeline configuration files
+  | `/etc/logstash/conf.d`
+  | `path.conf`
```

Comment:
My bad.  This should be `path.config` as the defaults in `logstash.yml` make plain:
```
# ------------ Pipeline Configuration Settings --------------
#
# Where to fetch the pipeline configuration for the main pipeline
#
# path.config:
```